react-components/my-app/src/Image.tsx

```diff
@@ -0,0 +1,8 @@
IdnYwfQhmfUSTD0BAh7Bh1If9vpUofepQTqr45a0IwisnDwcysNCUpUUlTKVmICSUkxqzRDZnYeGMNIHK5nMFJUI0BBjX09eWizMGJcFzokxsSHHmdjXQUUgZUqByMAXPM5dwAGJcmXswYmtMs9OgtAUEhKSpTvmIlRIhDB7SXuZcRQMXKQgBUwQoaXm4YXLMC8OavBWAyk8rBLGXQRlJWkplrkODD3LNOEWUn4hIJOYSHHMCCR1DuNixrRAvgBTSVFgP0gKAndxb3rXxWClK+RyksxUw0kEgkMJDvZjD0zgFqwznTlWAScq0csRzaAEOW2GrPScRKQwzvBgJh3aVPGkzVRJpwE/DzhaQQQkInMxBOcQwAIa7yKXzFhATq8gbln6dNqM8PiZQUoWQGBiHWIAd7iXba9B4jw60rOHjJWjESGUkgBrADKejTo9qfSZyBmYOwYmWvpGxjS4jSt+Lg5E/Ew1H4aypAV8pdISSlsxLAFIDmWOt0rxMIYSEpC/ihRzk5ciksMvVwXGxDazQKQ0hwGDu+ob0v79qRlJSDlfQkyzfq1eT1puGpP50qKWAGVUP1JBdwDEW6UtCOV8yAAzglLyU2F9X7BWj1eN4ks4YwifldIyhPylQURmErdTETpq8HT40eIqQ4OGoZJCXyBbBRbMAbyNTa7CsOPxamzuQFDLyw6UgJZmtCXLSQJd6BWAVZBlVISQoBSgUqeWiHBEahQmuXxZYs+ZOhDjMx0eHk7s5qboTJ44hOUgu8BMJsynuCQXIbz6Nmx8QOWDXLPEEmM0gMwAJJPmBQYmKkzlYFwmSdGdRJEu0sBJhgE1jzEhvP0rK7azBuYqcawwDTLb9rA0OFjFLsRMPqN272O4JFLCA4kbnp6tNCTU96vhiZMwLm3eBbsO1Ukd+wqgkGxNtvMvsL+lRHdrn0Fo3sP2vSNp4cJ5lZCUgPJLB4dRSxva0s8PWcmqSssz9daHrQXB5jv92oSfXeiTozk6hvZwXNWQAq7hrzqHZn8vWgwKGxdv50+96pqYcclIQVKKUklKSSwzM5A0JZP/iKBJ6/ZvQFpfuA/lo9SQJEH3D6eYMirXhsB72+7a/xVFib3NzGpvfoaAmaXAAjroGe99e9EMsiXidOsM+x8usCoGxDNeLTrVkMQ7zf+vKgKyF21v6gGiKCHBeA/q09iNeo3oSbR7fb0ScNT5ZBMENO7EXuBQBM0VVHhYCiISojcAkeoqUEYjFCQoOyi0gmxunlLF+WDZjrTgSxYkh5ATAcwSAYL2TFZ0kAEnPDMxAlUmDLEPPaJq8FZTl+VlcwliliQJd079eUzFVKVjqr4gmM5IZKdQGZ8stAMWbWIr1/4H8EwuKVihWBi4xTgryJwylGVYhOZ1BwRGsvBrxHDcWr4nxyUqUMQKKcR1lZKsxCgoEKD3zXeXrfwnimIlRXgfESopW5SslTKzFUpCVMEuSJgEnprLGVjWnECQAqxBfzzCOj/ANVs4TF+CvDxBhoLyBiZVoXLcwcAJkawRfbz446GcuA1zA0ADjUg9JLGm8d4krFX8TEKMynK1ZSE6sORwIGUBIADAQxNO6T+a6uH4sv4Rw/iK+GV/EWiGC7JVuWj7akcdxRWrMpasRag6iS5iOYqJNtTbtNcfE8QdIScsQCCdCVbyJOlJXxAGaWDOAp8ygSGZgzsc0lme8Ar9HMOl8SVAmQDImR+13NhfStPFeJnGT/qLdaQlIUdEpGUAlnYABgOut+GviCMzh1CCdQQ8ODsCXmwpAxS4LkAxmYhjD2ezgxLNvU3avw6juGF/Iam763gf95sXHLODLPrYEITI17tHcVgGOTDsxcF2y7keg6wKFSmSGUrMXzBmDQzF5fsLC9TdLmGvCxxOZZSUglJbNIsgC6XLy/lWXEf5nl2MHyBPkYqLxjkyuWdyNCQIN5LE1WFJjlAEke50D3jyqOqkXiC50fZnBkQLAs+ooisFIDqLbOwDOQHtdT9utLwcQpIWkDlIIJAIBdw4Lg2sXeaor+oaI1cX7d6DCttP+/LSrGj2+5HpQU9eHk5WcqSkg7BQCrXew9bvQEBzG7PuTc301/eqVhsC5EFmc815BZo/erxgoMlRDQrlIUOYAvBbMAQMsNYsXoEYxAYZdZKQTIIuQZmNixuHoC04byo5Q4eNC7lvKwoCwJ20fbrMVd7nsPpV4Q1L7JMM7i5MMz+1IKCyVPDk9AJ6WA9qoq7aaNbpamp5VQC7goKgN3BUkwxDGXHq9KSId209X+/OgGK5FKAZTZkg8wd3GYCCN2PmNKAKs4EdNLy0m9+1CS9y7yaiVe8dpeNqAIp9XkTHt39KvDIEkkbN97RTsRSGdlZ7GW/LJdyXzPp9aSsiSExF5Y3/Ys7x60BFZZZ7xa03m9vepltMH7ts/0qiQQNCIteXvqZ10FGQpLKciHDw8lJy7i/odqYDhoKiwBJ0F+sCjwVhMh3hrBmLkg6GI7+swMVKS+UnlIAzEEKaFuBYGW6N1pmBw6itIBE2UCwDcxOYjlYSdWoARjERyRHyoPu01VNwM2UNigCWGZQ12ECpQF4QkdcPEfqwxG9GHoKyZyzOdB5O7dnnvUqUAWh7j6GtPDKIUhi3Kr/APQ+kVKlVCrNiqO+g+gq8P5SdQQx2vVVKdA+EnEQDLqSC+zs3arwFHIoPDpju5P/ANU+g2qVKmglWIZLmbzeXneZoBUqUjSiVc+dSpQDePSBiKADB7ClKLlzJN+s1KlAHwg5h2P0NLFvMfvUqUBAoszwZbt/2fWopRLOSWgdBtUqUggNhpTgf9IjZaW6OFO3dh6CpUoCsaBhkQSku3/JY+kUOFZR1AjpzJq6lAFwSQVzPKsz0Qo03HDYWCoQrnLiDBDT0q6lAZcQwnsT7mfYelN4SVMZAQtn0ZCjHnPepUoBJUQS0dq15A2BAkl4vztO9SpQCAkfDJacwD+SqTUqUwtCyCCCQRYiDTkpGURqr2CalSgBTapUqUw//9k="
```

Comment:
😱 This is ok since it's only one image being exported, but if we had multiple img elements, embedded images would make your code hard to read.
here's the same image "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSSAODDbY5hnpV2EbuMbuTN_yT2BbcYvU9yzg&s"